Remove the Landmark detection web-app

yashksaini-coder opened the pull request:
### Description
removed all the files related to the Landmark detection web app

### Issue Resolved
<!-- This PR resolves #<issue-number>. -->
#134 

### Changes Made
Removing and format the dependencies to their original version before landmark detect web-app was integrated to it.

### Additional Details
<!-- Include any additional information that might be useful. -->

@yashasvini121 there's no alternative route or fix to integrate my landmark detect web app as it will cause dependencies error not only for my web app but to the other web-apps too. This will cause great dependency fails on the way, Hence removing this app is the best course of action

<!-- Don't remove this while submitting the issue -->
### Checklist
<!-- You can add more relevant points -->
- [X] This PR is submitted under **Hacktoberfest**.
- [X] This PR is submitted under **GirlScript Summer of Code (GSSoC-Extd)**.


---

I have removed all the files of my landmark detect web app, and also revert changes I have done. 
assign me this PR & add proper gssoc & hacktoberfest labels

yashksaini-coder commented:
@yashasvini121 review this PR and add labels, and close the #134 issue too after this.

yashasvini121 commented:
Hey, I have updated the requirements.txt, but unfortunately it's still giving errors, there is possibly some error with tensor-hub, a package not listed by you, but I had installed along with tensorrt.
the compatibility issues were with tensorflow, numpy and protobuf, so I removed their versions
![image](https://github.com/user-attachments/assets/e6821402-0e60-46c8-9f06-c0c8e1402a34)

Unfortunately I will have to remove the app for now, but you can look into this and try to fix it and create a new pr afterwards.

yashksaini-coder commented:
> Hey, I have updated the requirements.txt, but unfortunately it's still giving errors, there is possibly some error with tensor-hub, a package not listed by you, but I had installed along with tensorrt. the compatibility issues were with tensorflow, numpy and protobuf, so I removed their versions ![image](https://private-user-images.githubusercontent.com/100478608/377995358-e6821402-0e60-46c8-9f06-c0c8e1402a34.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjkyODIwNzAsIm5iZiI6MTcyOTI4MTc3MCwicGF0aCI6Ii8xMDA0Nzg2MDgvMzc3OTk1MzU4LWU2ODIxNDAyLTBlNjAtNDZjOC05ZjA2LWMwYzhlMTQwMmEzNC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDE4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAxOFQyMDAyNTBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lMTFhNDYzNTlkMjMxZmM0ZDIxZWI1YmVjMTE0N2RkZDAxNzk3Nzg5NjEwYzU5OWUyNzQ5M2QxNDlmZDQ1YzUzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.H62Q3SQNJSlmCFZxVLy0yI0jNq79GbI1CBfnGTdn1hk)
> 
> Unfortunately I will have to remove the app for now, but you can look into this and try to fix it and create a new pr afterwards.

yeah I get that, that's why the best decision is to remove such web app. 
Well better thing is you will learn from this & take this into account when new apps are merged

Also add proper labels on the issue before closing it. #134
